indexing-service/src/main/java/io/druid/indexing/overlord/TaskLockbox.java

```diff
@@ -66,6 +66,9 @@
 
   private static final EmittingLogger log = new EmittingLogger(TaskLockbox.class);
 
+  // Stores List of Active Tasks. TaskLockbox will only grant locks to active activeTasks.
+  private final Set<String> activeTasks = Sets.newHashSet();
```

Comment:
Does this need memory protection of the big lock to access or modify? If so it is probably worth calling out here.